src/helpers/userHelper.js

```diff
@@ -170,6 +171,14 @@ const userHelper = function () {
           ? `<a href="${mediaUrl}">${mediaUrl}</a>`
           : 'Not provided!';
 
+        const googleDocLinkValue = adminLinks.length > 0
```

Comment:
Since `adminLinks` is an array, I would add a check to ensure adminLinks is not undefined or null before attempting to access its length property or performing a find operation. This can prevent potential runtime errors if adminLinks is ever undefined.

```js
adminLinks?.length > 0 
```

Reply:
Hello, @FyisFe! I agree with your suggestion. Will make the update. Thank you!